de.dlr.sc.virsat.model.ui/src/de/dlr/sc/virsat/model/ui/provider/ActiveConceptsLabelProvider.java

```diff
@@ -25,10 +25,18 @@
 	public String getText(Object element) {
 		if (element instanceof ActiveConceptConfigurationElement) {
 			ActiveConceptConfigurationElement acce = (ActiveConceptConfigurationElement) element;
-			return acce.getId() + " [" + acce.getVersion() + "]";
+			String acceDisplayName = acce.getDisplayName();
+			
+			if(acceDisplayName == null) {
+				return acce.getId() + " [" + acce.getVersion() + "]";
+				
+			} else {
+				return acceDisplayName + " – " + acce.getId() + " [" + acce.getVersion() + "]";
+			}
+			
 		} else if (element instanceof Concept) {
 			Concept concept = (Concept) element;
-			return concept.getName() + " [" + concept.getVersion() + "]";
+			return concept.getDisplayName() + " [" + concept.getVersion() + "]";

```

Comment:
I think in the case that we have the concept already loaded it should also have the same structure of "<Displayname> - <FullQualifiedName>[<Version>]"... As it is now it doesn't show its name anymore